.gitignore

```diff
@@ -11,3 +11,4 @@ etl-daemon.js
 coverage
 conf/config-bk.json
 .DS_Store
+package-lock.json
```

Comment:
We shouldn't ignore this file. It ensures everyone has the same version of dependencies.

Reply:
Noted.